msi/rust.wxs

```diff
@@ -149,6 +154,10 @@
         <!-- Specifies a single cab file to be embedded in the installer's .msi. -->
         <MediaTemplate EmbedCab="yes" CompressionLevel="high" />
 
+        <!-- Send a WM_SETTINGCHANGE message to tell processes like explorer to update their
+             environments so any new command prompts get the updated %PATH% -->
+        <CustomActionRef Id="WixBroadcastEnvironmentChange" />
```

Comment:
Ahh, ok. Looks like it's indeed a [Windows 10 bug](https://social.technet.microsoft.com/Forums/en-US/4db362ce-ce9c-49c8-991c-c38f5b740cb7/msi-bug-in-windows-10-path-env-var-not-updated-by-msi-after-sw-install-finishes-until-user-log?forum=win10itprogeneral). 
Though seeing that WixBroadcastEnvironmentChange [still uses a broadcast](https://github.com/wixtoolset/wix3/blob/master/src/ext/ca/wixca/dll/BroadcastSettingChange.cpp), it's unclear to me why would it have better luck with that than the Windows Installer...